### ISSUE TITLE
images from folder "set as available offline" are added to camera roll

### ISSUE THREAD

**Issue by vonProteus** [nextcloud/ios#1392]
### Expected behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
they shouldn't

### Actual behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
when i set folder as available offline images and movies from that folder are added to camera roll

### Steps to reproduce     
1. set folder as available offline 
2. wait for sync 
3. they appeared in camera roll 

### Reasoning or why should it be changed/implemented?
i think it’s a bug or missing toggle for that kind of behavior in settings

#### iOS version
13.6.1

### App version
3.0.4.11

### Server configuration

**Operating system:**
Raspberry Pi OS
kernel: 5.4.51-v8+ aarch64 
with docker 

**Web server:**
docker image nextcloud:stable@sha256:30deb5ee4fc72199f0efe2ded1a426f6ec288c092a56ebaa82194e94501b6642

**Database:**
docker image  postgres:11@sha256:23e692b619ddc37c656acfbcc41829bd9f0ee9e940df3a8bf04da6a5c9d7511e

**PHP version:**
docker image nextcloud:stable@sha256:30deb5ee4fc72199f0efe2ded1a426f6ec288c092a56ebaa82194e94501b6642

**Nextcloud version:** (see Nextcloud admin page)
docker image nextcloud:stable@sha256:30deb5ee4fc72199f0efe2ded1a426f6ec288c092a56ebaa82194e94501b6642
18.0.7


my whole stack look like this
```
version: '3.6'

services:
  db:
    image: postgres:11
    volumes:
    - db_data:/var/lib/postgresql/data
    environment:
      - POSTGRES_USER=nextcloud
      - POSTGRES_PASSWORD=nextcloud
      - POSTGRES_DB=notthis
    networks:
      - internal
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
        delay: 5s

  redis:
    image: redis:alpine
    volumes:
      - redis_data:/data
    networks:
      - internal
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
        delay: 5s

  cron:
    image: nextcloud:stable
    volumes:
      - nextcloud:/var/www/html
      - apps:/var/www/html/custom_apps
      - config:/var/www/html/config
      - data:/var/www/html/data
    entrypoint: /cron.sh
    networks:
      - internal
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
        delay: 5s

  nextcloud:
    image: nextcloud:stable
    environment:
      - POSTGRES_USER=nextcloud
      - POSTGRES_PASSWORD=nextcloud
      - POSTGRES_DB=nextcloud
      - POSTGRES_HOST=db:5432
      - REDIS_HOST=redis
    volumes:
      - nextcloud:/var/www/html
      - apps:/var/www/html/custom_apps
      - config:/var/www/html/config
      - data:/var/www/html/data
    networks:
      - internal
      - external
      - traefik-net
    deploy:
      mode: replicated
      replicas: 1
      restart_policy:
        condition: on-failure
        delay: 5s
      labels:
        traefik.enable: "true"
        traefik.frontend.rule: "Host:${HOST}"
        traefik.port: 80
        traefik.docker.network: traefik-net
        traefik.frontend.headers.STSSeconds: 63072000
        traefik.frontend.headers.STSIncludeSubdomains: "true"
        traefik.frontend.headers.STSPreload: "true"
        traefik.frontend.redirect.permanent: 'true'
        traefik.frontend.redirect.regex: https://(.*)/.well-known/(card|cal)dav
        traefik.frontend.redirect.replacement: https://$$1/remote.php/dav/


volumes:
  db_data:
  redis_data:
  nextcloud:
  apps:
  config:
  data:


networks:
  external:
  internal:
    driver: overlay
    internal: true
  traefik-net:
    external: true

### Environment variables
# HOST
```

**Comment by marinofaggiana** [comment]
Fixed. 3.0.6

**Comment by vonProteus** [comment]
confirm fix in 3.0.6.8
